pkg/apis/kops/v1alpha2/componentconfig.go

```diff
@@ -610,6 +610,12 @@ type KubeSchedulerConfig struct {
 	// which has been supported as far back as Kubernetes 1.7. The default depends on the version and the cloud provider
 	// as outlined: https://kubernetes.io/docs/concepts/storage/storage-limits/
 	MaxPersistentVolumes *int32 `json:"maxPersistentVolumes,omitempty"`
+	// Qps sets the maximum qps to send to apiserver after the burst quota is exhausted
+	Qps *resource.Quantity `json:"qps,omitempty" configfile:"ClientConnection.QPS"`
+	// Burst sets the maximum qps to send to apiserver after the burst quota is exhausted
+	Burst *int32 `json:"burst,omitempty" configfile:"ClientConnection.Burst"`
```

Comment:
Would it make sense to change one of the e2e tests to fill in these two values? That would make it more likely we'd detect k/k changing the name or type of these settings.